app/views/tasks/index.html.erb

```diff
@@ -1,2 +1,11 @@
-<h1>Tasks#index</h1>
-<p>Find me in app/views/tasks/index.html.erb</p>
+<div class="row">
+  <div class='col-md-4'>
+    <%= render 'done_tasks', task: @tasks_done %>
+  </div>
+  <div class='col-md-4'>
+    <%= render 'in_progress_tasks', task: @tasks_started %>
+  </div>
+  <div class='col-md-4'>
+    <%= render 'new_tasks', task: @tasks_not_started %>
```

Comment:
the same as [above](https://github.com/dKotetunov/TasksManager/pull/2/files#r27945300)